ui/page/governance/proposal_details_page.go

```diff
@@ -120,12 +133,56 @@ func (pg *ProposalDetails) HandleUserInteractions() {
 	}
 
 	for pg.viewInPoliteiaBtn.Clicked() {
-		host := "https://proposals.decred.org/record/"
+		host := "https://proposals.decred.org/record/" + pg.proposal.Token
 		if pg.WL.MultiWallet.NetType() == dcrlibwallet.Testnet3 {
-			host = "https://test-proposals.decred.org/record/"
+			host = "https://test-proposals.decred.org/record/" + pg.proposal.Token
 		}
 
-		components.GoToURL(host + pg.proposal.Token)
+		info := modal.NewInfoModal(pg.Load).
+			Title("View on Politeia").
+			Body("Copy the link below to your browser.").
```

Comment:
i'm not sure this text is clear enough, doesn't seem complete to me

"paste the link below in your browser"

i don't know something more crisp